src/mscorlib/mscorlib.shared.sources.props

```diff
@@ -1119,7 +1143,6 @@
     <GenericsSources Include="$(BclSourcesRoot)\System\Collections\Concurrent\PartitionerStatic.cs" />
   </ItemGroup>
   <ItemGroup>
-    <SafehandleSources Include="$(BclSourcesRoot)\Microsoft\Win32\SafeHandles\SafeFileHandle.cs" />
```

Comment:
It would be useful to explicitly ifdef out all existing uses of SafeFileHandle for CoreCLR - because of it goes to CoreCLR PAL. The CoreCLR PAL file handles are different from the corefx file handles.